pygradle-plugin/src/integTest/groovy/com/linkedin/gradle/python/plugin/PythonPluginIntegrationTest.groovy

```diff
@@ -96,4 +96,43 @@ class PythonPluginIntegrationTest extends Specification {
         result.task(':check').outcome == TaskOutcome.SUCCESS
         result.task(':build').outcome == TaskOutcome.SUCCESS
     }
+
+    def "can build v3 library"() {
+        given:
+        testProjectDir.buildFile << """
+        |plugins {
+        |    id 'com.linkedin.python'
+        |}
+        |
+        |python {
+        | details {
+        |   pythonVersion = 3.5
```

Comment:
I know, but the integration tests show the usage pattern too.
And we would prefer to keep that pattern match the intended usage.